Consolidate RuntimeApplicationChecks in WTF

aestes opened the pull request:
#### 6cdbb2d7e7049de43514c56dc660f53166ebc7ce
<pre>
Consolidate RuntimeApplicationChecks in WTF
<a href="https://bugs.webkit.org/show_bug.cgi?id=281014">https://bugs.webkit.org/show_bug.cgi?id=281014</a>
<a href="https://rdar.apple.com/137460460">rdar://137460460</a>

Reviewed by Brent Fulgham.

RuntimeApplicationChecks related to SDK-aligned behaviors lived in WTF, but checks related to
bundle identifiers, etc. lived in WebCore. To assist in a follow-on change that will require
checking the application bundle identifier when computing SDK-aligned behaviors, this change
consolidates RuntimeApplicationChecks(Cocoa) in WTF.

* Source/WTF/WTF.xcodeproj/project.pbxproj:
* Source/WTF/wtf/RuntimeApplicationChecks.cpp:
  Renamed from Source/WebCore/platform/RuntimeApplicationChecks.cpp.
* Source/WTF/wtf/RuntimeApplicationChecks.h:
  Renamed from Source/WebCore/platform/RuntimeApplicationChecks.h.
* Source/WTF/wtf/cocoa/RuntimeApplicationChecksCocoa.h:
  Moved Cocoa-specific code from Source/WebCore/platform/RuntimeApplicationChecks.h.
* Source/WTF/wtf/cocoa/RuntimeApplicationChecksCocoa.mm:
  Renamed from Source/WTF/wtf/cocoa/RuntimeApplicationChecksCocoa.cpp.
* Source/WebCore/Modules/geolocation/Geolocation.cpp:
* Source/WebCore/Sources.txt:
* Source/WebCore/SourcesCocoa.txt:
* Source/WebCore/WebCore.xcodeproj/project.pbxproj:
* Source/WebCore/accessibility/AccessibilityObject.h:
* Source/WebCore/accessibility/ios/WebAccessibilityObjectWrapperIOS.mm:
* Source/WebCore/accessibility/isolatedtree/AXIsolatedTree.h:
* Source/WebCore/accessibility/mac/WebAccessibilityObjectWrapperMac.mm:
* Source/WebCore/bindings/js/JSDOMWindowBase.cpp:
* Source/WebCore/bindings/js/JSLocationCustom.cpp:
* Source/WebCore/bindings/js/ScriptController.cpp:
* Source/WebCore/dom/Document.cpp:
* Source/WebCore/dom/Element.cpp:
* Source/WebCore/dom/ScriptDisallowedScope.h:
* Source/WebCore/dom/ScriptElement.cpp:
* Source/WebCore/editing/mac/EditorMac.mm:
* Source/WebCore/html/HTMLAnchorElement.cpp:
* Source/WebCore/html/HTMLMediaElement.cpp:
* Source/WebCore/html/HTMLObjectElement.cpp:
* Source/WebCore/html/MediaElementSession.cpp:
* Source/WebCore/inspector/agents/InspectorTimelineAgent.cpp:
* Source/WebCore/loader/CrossOriginAccessControl.cpp:
* Source/WebCore/loader/DocumentThreadableLoader.cpp:
* Source/WebCore/loader/FrameLoader.cpp:
* Source/WebCore/loader/SubresourceLoader.cpp:
* Source/WebCore/loader/cache/CachedResourceLoader.cpp:
* Source/WebCore/loader/cache/CachedScript.cpp:
* Source/WebCore/page/EventHandler.cpp:
* Source/WebCore/page/LocalDOMWindow.cpp:
* Source/WebCore/page/OriginAccessPatterns.cpp:
* Source/WebCore/page/Quirks.cpp:
* Source/WebCore/page/SecurityOrigin.cpp:
* Source/WebCore/page/SettingsBase.cpp:
* Source/WebCore/page/mac/EventHandlerMac.mm:
* Source/WebCore/platform/LegacySchemeRegistry.cpp:
* Source/WebCore/platform/RuntimeApplicationChecks.h: Removed.
* Source/WebCore/platform/Timer.cpp:
* Source/WebCore/platform/audio/ios/MediaSessionManagerIOS.mm:
* Source/WebCore/platform/cocoa/NetworkExtensionContentFilter.mm:
* Source/WebCore/platform/cocoa/RuntimeApplicationChecksCocoa.mm: Removed.
* Source/WebCore/platform/cocoa/TextRecognitionResultCocoa.mm:
* Source/WebCore/platform/gamepad/cocoa/GameControllerGamepad.mm:
* Source/WebCore/platform/graphics/FontCascade.h:
* Source/WebCore/platform/graphics/angle/GraphicsContextGLANGLE.cpp:
* Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateAVFoundationObjC.mm:
* Source/WebCore/platform/graphics/cg/ImageBufferCGBackend.cpp:
* Source/WebCore/platform/graphics/cg/PDFDocumentImage.h:
* Source/WebCore/platform/graphics/cocoa/GraphicsContextGLCocoa.mm:
* Source/WebCore/platform/graphics/coretext/FontCascadeCoreText.cpp:
* Source/WebCore/platform/graphics/mac/LegacyDisplayRefreshMonitorMac.cpp:
* Source/WebCore/platform/graphics/mac/controls/ControlMac.mm:
* Source/WebCore/platform/ios/PlatformPasteboardIOS.mm:
* Source/WebCore/platform/ios/UIViewControllerUtilities.mm:
* Source/WebCore/platform/ios/UserAgentIOS.mm:
* Source/WebCore/platform/ios/VideoPresentationInterfaceIOS.mm:
* Source/WebCore/platform/ios/wak/WebCoreThread.mm:
* Source/WebCore/platform/mac/WidgetMac.mm:
* Source/WebCore/platform/mediastream/ios/MediaCaptureStatusBarManager.mm:
* Source/WebCore/platform/mediastream/mac/RealtimeMediaSourceCenterMac.mm:
* Source/WebCore/platform/network/DNS.cpp:
* Source/WebCore/platform/network/NetworkStorageSession.cpp:
* Source/WebCore/platform/network/cocoa/ProtectionSpaceCocoa.mm:
* Source/WebCore/platform/network/cocoa/ResourceRequestCocoa.mm:
* Source/WebCore/platform/network/mac/ResourceHandleMac.mm:
* Source/WebCore/rendering/RenderBox.cpp:
* Source/WebCore/rendering/RenderBoxModelObject.cpp:
* Source/WebCore/rendering/RenderLayerBacking.cpp:
* Source/WebCore/xml/XMLHttpRequest.cpp:
* Source/WebKit/GPUProcess/GPUProcess.cpp:
* Source/WebKit/GPUProcess/GPUProcess.h:
* Source/WebKit/ModelProcess/ModelProcess.cpp:
* Source/WebKit/ModelProcess/ModelProcess.h:
* Source/WebKit/NetworkProcess/NetworkProcess.cpp:
* Source/WebKit/NetworkProcess/NetworkProcess.h:
* Source/WebKit/NetworkProcess/NetworkSession.cpp:
* Source/WebKit/NetworkProcess/PrivateClickMeasurement/PrivateClickMeasurementManager.cpp:
* Source/WebKit/NetworkProcess/cocoa/NetworkProcessCocoa.mm:
* Source/WebKit/NetworkProcess/cocoa/NetworkSessionCocoa.mm:
* Source/WebKit/Platform/IPC/HandleMessage.h:
* Source/WebKit/Platform/ios/PaymentAuthorizationController.mm:
* Source/WebKit/Shared/AuxiliaryProcess.h:
* Source/WebKit/Shared/Cocoa/ArgumentCodersCocoa.mm:
* Source/WebKit/Shared/Cocoa/AuxiliaryProcessCocoa.mm:
* Source/WebKit/Shared/Cocoa/CoreIPCSecureCoding.mm:
* Source/WebKit/Shared/Cocoa/DefaultWebBrowserChecks.mm:
* Source/WebKit/Shared/EntryPointUtilities/Cocoa/XPCService/XPCEndpointMessages.mm:
* Source/WebKit/Shared/RemoteLayerTree/RemoteLayerTreeTransaction.mm:
* Source/WebKit/Shared/WebPreferencesDefaultValues.cpp:
* Source/WebKit/Shared/ios/NativeWebKeyboardEventIOS.mm:
* Source/WebKit/Shared/ios/WebPreferencesDefaultValuesIOS.mm:
* Source/WebKit/Shared/mac/AuxiliaryProcessMac.mm:
* Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm:
* Source/WebKit/UIProcess/API/Cocoa/WKWebViewConfiguration.mm:
* Source/WebKit/UIProcess/API/Cocoa/WKWebViewTesting.mm:
* Source/WebKit/UIProcess/API/ios/WKWebViewIOS.mm:
* Source/WebKit/UIProcess/Cocoa/SOAuthorization/SOAuthorizationSession.mm:
* Source/WebKit/UIProcess/Cocoa/UserMediaPermissionRequestManagerProxy.mm:
* Source/WebKit/UIProcess/Cocoa/WKShareSheet.mm:
* Source/WebKit/UIProcess/Cocoa/WebProcessPoolCocoa.mm:
* Source/WebKit/UIProcess/Cocoa/WebProcessProxyCocoa.mm:
* Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm:
* Source/WebKit/UIProcess/GPU/GPUProcessProxy.cpp:
* Source/WebKit/UIProcess/Inspector/WebInspectorUtilities.cpp:
* Source/WebKit/UIProcess/Launcher/cocoa/ProcessLauncherCocoa.mm:
* Source/WebKit/UIProcess/Model/ModelProcessProxy.cpp:
* Source/WebKit/UIProcess/Network/NetworkProcessProxyCocoa.mm:
* Source/WebKit/UIProcess/RemoteLayerTree/RemoteScrollingCoordinatorProxy.cpp:
* Source/WebKit/UIProcess/WebPageProxy.cpp:
* Source/WebKit/UIProcess/WebProcessPool.cpp:
* Source/WebKit/UIProcess/WebsiteData/Cocoa/WebsiteDataStoreCocoa.mm:
* Source/WebKit/UIProcess/ios/WKBaseScrollView.mm:
* Source/WebKit/UIProcess/ios/WKContentView.mm:
* Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm:
* Source/WebKit/UIProcess/ios/WebPageProxyIOS.mm:
* Source/WebKit/UIProcess/mac/WebPageProxyMac.mm:
* Source/WebKit/WebProcess/Network/WebLoaderStrategy.cpp:
* Source/WebKit/WebProcess/WebCoreSupport/WebLocalFrameLoaderClient.cpp:
* Source/WebKit/WebProcess/WebPage/ios/WebPageIOS.mm:
* Source/WebKit/WebProcess/WebPage/mac/WebPageMac.mm:
* Source/WebKit/WebProcess/WebProcess.cpp:
* Source/WebKit/WebProcess/WebProcess.h:
* Source/WebKit/WebProcess/cocoa/WebProcessCocoa.mm:
* Source/WebKitLegacy/WebCoreSupport/WebResourceLoadScheduler.cpp:
* Source/WebKitLegacy/ios/WebCoreSupport/WebChromeClientIOS.mm:
* Source/WebKitLegacy/mac/Misc/WebDownload.mm:
* Source/WebKitLegacy/mac/Plugins/WebPluginController.mm:
* Source/WebKitLegacy/mac/WebCoreSupport/WebContextMenuClient.mm:
* Source/WebKitLegacy/mac/WebCoreSupport/WebFrameLoaderClient.mm:
* Source/WebKitLegacy/mac/WebView/WebFrame.mm:
* Source/WebKitLegacy/mac/WebView/WebHTMLView.mm:
* Source/WebKitLegacy/mac/WebView/WebImmediateActionController.mm:
* Source/WebKitLegacy/mac/WebView/WebPDFView.mm:
* Source/WebKitLegacy/mac/WebView/WebPreferences.mm:
* Source/WebKitLegacy/mac/WebView/WebPreferencesDefaultValues.mm:
* Source/WebKitLegacy/mac/WebView/WebResource.mm:
* Source/WebKitLegacy/mac/WebView/WebView.mm:
* Source/WebKitLegacy/mac/WebView/WebViewRenderingUpdateScheduler.mm:
* Tools/TestWebKitAPI/Tests/WebCore/cocoa/TestGraphicsContextGLCocoa.mm:
* Tools/TestWebKitAPI/Tests/WebKitCocoa/InAppBrowserPrivacy.mm:
* Tools/TestWebKitAPI/WebCoreTestUtilities.h:
  Fixed imports and namespace qualifiers to reflect new location of RuntimeApplicationChecks.h.

Canonical link: <a href="https://commits.webkit.org/284864@main">https://commits.webkit.org/284864@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/879872ee9b531dc89d813a0e151ab264e3c63a1e

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [❌ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/70705 "3 style errors") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/50115 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/23474 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/74798 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/21887 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/72821 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/57913 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/21726 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/55994 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/14460 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/73771 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/45570 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/60961 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/36445 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/42227 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/18395 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/20247 "Built successfully") | 
| [✅ 🛠 🧪 jsc](https://ews-build.webkit.org/#/builders/20/builds/63827 "Built successfully and passed tests") | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/64161 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/18756 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/76516 "Built successfully") | 
| [✅ 🛠 🧪 jsc-arm64](https://ews-build.webkit.org/#/builders/12/builds/69954 "Built successfully and passed tests") | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/87/builds/14935 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/62/builds/17958 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/63722 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/86/builds/14979 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/61024 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/63658 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/15666 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/88/builds/11718 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/119/builds/5367 "Passed tests") | [✅ 🛠 jsc-armv7](https://ews-build.webkit.org/#/builders/35/builds/91735 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/45916 "Built successfully") | [✅ 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/687 "Built successfully") | [✅ 🧪 jsc-armv7-tests](https://ews-build.webkit.org/#/builders/25/builds/20001 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/46988 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/48269 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/46730 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

brentfulgham commented on Source/WebCore/editing/mac/EditorMac.mm:
```diff
@@ -54,14 +54,14 @@
 #import <AppKit/AppKit.h>
 #import <UniformTypeIdentifiers/UniformTypeIdentifiers.h>
 #import <wtf/RetainPtr.h>
+#import <wtf/RuntimeApplicationChecks.h>
```
Sort of seems like we would want the Cocoa version, but I guess not!

aestes replied:
Yeah, the non-Cocoa one is sufficient.

brentfulgham commented on Source/WebCore/platform/graphics/cocoa/GraphicsContextGLCocoa.mm:
```diff
@@ -37,12 +37,12 @@
 #import <pal/spi/cg/CoreGraphicsSPI.h>
 #import <pal/spi/cocoa/MetalSPI.h>
 #import <wtf/BlockObjCExceptions.h>
+#import <wtf/RuntimeApplicationChecks.h>
```
Why not the cocoa one?

aestes replied:
I could, but the non-Cocoa one is sufficient.

brentfulgham commented on Source/WebCore/platform/graphics/coretext/FontCoreText.cpp:
```diff
@@ -31,6 +31,7 @@
 #include "FloatRect.h"
 #include "FontCache.h"
 #include "FontCascade.h"
+#include "FontCustomPlatformData.h"
```
Unified builds are fun.

brentfulgham commented on Source/WebKit/UIProcess/API/Cocoa/WKWebViewConfiguration.mm:
```diff
@@ -43,7 +43,6 @@
 #import "WebURLSchemeHandlerCocoa.h"
 #import "_WKApplicationManifestInternal.h"
 #import "_WKVisitedLinkStoreInternal.h"
-#import <WebCore/RuntimeApplicationChecks.h>
```
Seems weird that we don't need to include a `RuntimeApplicationChecks.h` at all, especially when you call them below.

This probably breaks a non-unified build -- maybe you should just add the `<wtf/cocoa/RuntimeApplicationChecks.h>` file?

aestes replied:
`RuntimeApplicationChecksCocoa.h` was already imported on line 57, so I'm just removing a duplicate import.

brentfulgham reviewed (approved):
Looks good to me. I had a few minor nits about possibly breaking non-unified builds.

aestes committed:
Consolidate RuntimeApplicationChecks in WTF
https://bugs.webkit.org/show_bug.cgi?id=281014
rdar://137460460

Reviewed by Brent Fulgham.

RuntimeApplicationChecks related to SDK-aligned behaviors lived in WTF, but checks related to
bundle identifiers, etc. lived in WebCore. To assist in a follow-on change that will require
checking the application bundle identifier when computing SDK-aligned behaviors, this change
consolidates RuntimeApplicationChecks(Cocoa) in WTF.

* Source/WTF/WTF.xcodeproj/project.pbxproj:
* Source/WTF/wtf/RuntimeApplicationChecks.cpp:
  Renamed from Source/WebCore/platform/RuntimeApplicationChecks.cpp.
* Source/WTF/wtf/RuntimeApplicationChecks.h:
  Renamed from Source/WebCore/platform/RuntimeApplicationChecks.h.
* Source/WTF/wtf/cocoa/RuntimeApplicationChecksCocoa.h:
  Moved Cocoa-specific code from Source/WebCore/platform/RuntimeApplicationChecks.h.
* Source/WTF/wtf/cocoa/RuntimeApplicationChecksCocoa.mm:
  Renamed from Source/WTF/wtf/cocoa/RuntimeApplicationChecksCocoa.cpp.
* Source/WebCore/Modules/geolocation/Geolocation.cpp:
* Source/WebCore/Sources.txt:
* Source/WebCore/SourcesCocoa.txt:
* Source/WebCore/WebCore.xcodeproj/project.pbxproj:
* Source/WebCore/accessibility/AccessibilityObject.h:
* Source/WebCore/accessibility/ios/WebAccessibilityObjectWrapperIOS.mm:
* Source/WebCore/accessibility/isolatedtree/AXIsolatedTree.h:
* Source/WebCore/accessibility/mac/WebAccessibilityObjectWrapperMac.mm:
* Source/WebCore/bindings/js/JSDOMWindowBase.cpp:
* Source/WebCore/bindings/js/JSLocationCustom.cpp:
* Source/WebCore/bindings/js/ScriptController.cpp:
* Source/WebCore/dom/Document.cpp:
* Source/WebCore/dom/Element.cpp:
* Source/WebCore/dom/ScriptDisallowedScope.h:
* Source/WebCore/dom/ScriptElement.cpp:
* Source/WebCore/editing/mac/EditorMac.mm:
* Source/WebCore/html/HTMLAnchorElement.cpp:
* Source/WebCore/html/HTMLMediaElement.cpp:
* Source/WebCore/html/HTMLObjectElement.cpp:
* Source/WebCore/html/MediaElementSession.cpp:
* Source/WebCore/inspector/agents/InspectorTimelineAgent.cpp:
* Source/WebCore/loader/CrossOriginAccessControl.cpp:
* Source/WebCore/loader/DocumentThreadableLoader.cpp:
* Source/WebCore/loader/FrameLoader.cpp:
* Source/WebCore/loader/SubresourceLoader.cpp:
* Source/WebCore/loader/cache/CachedResourceLoader.cpp:
* Source/WebCore/loader/cache/CachedScript.cpp:
* Source/WebCore/page/EventHandler.cpp:
* Source/WebCore/page/LocalDOMWindow.cpp:
* Source/WebCore/page/OriginAccessPatterns.cpp:
* Source/WebCore/page/Quirks.cpp:
* Source/WebCore/page/SecurityOrigin.cpp:
* Source/WebCore/page/SettingsBase.cpp:
* Source/WebCore/page/mac/EventHandlerMac.mm:
* Source/WebCore/platform/LegacySchemeRegistry.cpp:
* Source/WebCore/platform/RuntimeApplicationChecks.h: Removed.
* Source/WebCore/platform/Timer.cpp:
* Source/WebCore/platform/audio/ios/MediaSessionManagerIOS.mm:
* Source/WebCore/platform/cocoa/NetworkExtensionContentFilter.mm:
* Source/WebCore/platform/cocoa/RuntimeApplicationChecksCocoa.mm: Removed.
* Source/WebCore/platform/cocoa/TextRecognitionResultCocoa.mm:
* Source/WebCore/platform/gamepad/cocoa/GameControllerGamepad.mm:
* Source/WebCore/platform/graphics/FontCascade.h:
* Source/WebCore/platform/graphics/angle/GraphicsContextGLANGLE.cpp:
* Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateAVFoundationObjC.mm:
* Source/WebCore/platform/graphics/cg/ImageBufferCGBackend.cpp:
* Source/WebCore/platform/graphics/cg/PDFDocumentImage.h:
* Source/WebCore/platform/graphics/cocoa/GraphicsContextGLCocoa.mm:
* Source/WebCore/platform/graphics/coretext/FontCascadeCoreText.cpp:
* Source/WebCore/platform/graphics/mac/LegacyDisplayRefreshMonitorMac.cpp:
* Source/WebCore/platform/graphics/mac/controls/ControlMac.mm:
* Source/WebCore/platform/ios/PlatformPasteboardIOS.mm:
* Source/WebCore/platform/ios/UIViewControllerUtilities.mm:
* Source/WebCore/platform/ios/UserAgentIOS.mm:
* Source/WebCore/platform/ios/VideoPresentationInterfaceIOS.mm:
* Source/WebCore/platform/ios/wak/WebCoreThread.mm:
* Source/WebCore/platform/mac/WidgetMac.mm:
* Source/WebCore/platform/mediastream/ios/MediaCaptureStatusBarManager.mm:
* Source/WebCore/platform/mediastream/mac/RealtimeMediaSourceCenterMac.mm:
* Source/WebCore/platform/network/DNS.cpp:
* Source/WebCore/platform/network/NetworkStorageSession.cpp:
* Source/WebCore/platform/network/cocoa/ProtectionSpaceCocoa.mm:
* Source/WebCore/platform/network/cocoa/ResourceRequestCocoa.mm:
* Source/WebCore/platform/network/mac/ResourceHandleMac.mm:
* Source/WebCore/rendering/RenderBox.cpp:
* Source/WebCore/rendering/RenderBoxModelObject.cpp:
* Source/WebCore/rendering/RenderLayerBacking.cpp:
* Source/WebCore/xml/XMLHttpRequest.cpp:
* Source/WebKit/GPUProcess/GPUProcess.cpp:
* Source/WebKit/GPUProcess/GPUProcess.h:
* Source/WebKit/ModelProcess/ModelProcess.cpp:
* Source/WebKit/ModelProcess/ModelProcess.h:
* Source/WebKit/NetworkProcess/NetworkProcess.cpp:
* Source/WebKit/NetworkProcess/NetworkProcess.h:
* Source/WebKit/NetworkProcess/NetworkSession.cpp:
* Source/WebKit/NetworkProcess/PrivateClickMeasurement/PrivateClickMeasurementManager.cpp:
* Source/WebKit/NetworkProcess/cocoa/NetworkProcessCocoa.mm:
* Source/WebKit/NetworkProcess/cocoa/NetworkSessionCocoa.mm:
* Source/WebKit/Platform/IPC/HandleMessage.h:
* Source/WebKit/Platform/ios/PaymentAuthorizationController.mm:
* Source/WebKit/Shared/AuxiliaryProcess.h:
* Source/WebKit/Shared/Cocoa/ArgumentCodersCocoa.mm:
* Source/WebKit/Shared/Cocoa/AuxiliaryProcessCocoa.mm:
* Source/WebKit/Shared/Cocoa/CoreIPCSecureCoding.mm:
* Source/WebKit/Shared/Cocoa/DefaultWebBrowserChecks.mm:
* Source/WebKit/Shared/EntryPointUtilities/Cocoa/XPCService/XPCEndpointMessages.mm:
* Source/WebKit/Shared/RemoteLayerTree/RemoteLayerTreeTransaction.mm:
* Source/WebKit/Shared/WebPreferencesDefaultValues.cpp:
* Source/WebKit/Shared/ios/NativeWebKeyboardEventIOS.mm:
* Source/WebKit/Shared/ios/WebPreferencesDefaultValuesIOS.mm:
* Source/WebKit/Shared/mac/AuxiliaryProcessMac.mm:
* Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm:
* Source/WebKit/UIProcess/API/Cocoa/WKWebViewConfiguration.mm:
* Source/WebKit/UIProcess/API/Cocoa/WKWebViewTesting.mm:
* Source/WebKit/UIProcess/API/ios/WKWebViewIOS.mm:
* Source/WebKit/UIProcess/Cocoa/SOAuthorization/SOAuthorizationSession.mm:
* Source/WebKit/UIProcess/Cocoa/UserMediaPermissionRequestManagerProxy.mm:
* Source/WebKit/UIProcess/Cocoa/WKShareSheet.mm:
* Source/WebKit/UIProcess/Cocoa/WebProcessPoolCocoa.mm:
* Source/WebKit/UIProcess/Cocoa/WebProcessProxyCocoa.mm:
* Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm:
* Source/WebKit/UIProcess/GPU/GPUProcessProxy.cpp:
* Source/WebKit/UIProcess/Inspector/WebInspectorUtilities.cpp:
* Source/WebKit/UIProcess/Launcher/cocoa/ProcessLauncherCocoa.mm:
* Source/WebKit/UIProcess/Model/ModelProcessProxy.cpp:
* Source/WebKit/UIProcess/Network/NetworkProcessProxyCocoa.mm:
* Source/WebKit/UIProcess/RemoteLayerTree/RemoteScrollingCoordinatorProxy.cpp:
* Source/WebKit/UIProcess/WebPageProxy.cpp:
* Source/WebKit/UIProcess/WebProcessPool.cpp:
* Source/WebKit/UIProcess/WebsiteData/Cocoa/WebsiteDataStoreCocoa.mm:
* Source/WebKit/UIProcess/ios/WKBaseScrollView.mm:
* Source/WebKit/UIProcess/ios/WKContentView.mm:
* Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm:
* Source/WebKit/UIProcess/ios/WebPageProxyIOS.mm:
* Source/WebKit/UIProcess/mac/WebPageProxyMac.mm:
* Source/WebKit/WebProcess/Network/WebLoaderStrategy.cpp:
* Source/WebKit/WebProcess/WebCoreSupport/WebLocalFrameLoaderClient.cpp:
* Source/WebKit/WebProcess/WebPage/ios/WebPageIOS.mm:
* Source/WebKit/WebProcess/WebPage/mac/WebPageMac.mm:
* Source/WebKit/WebProcess/WebProcess.cpp:
* Source/WebKit/WebProcess/WebProcess.h:
* Source/WebKit/WebProcess/cocoa/WebProcessCocoa.mm:
* Source/WebKitLegacy/WebCoreSupport/WebResourceLoadScheduler.cpp:
* Source/WebKitLegacy/ios/WebCoreSupport/WebChromeClientIOS.mm:
* Source/WebKitLegacy/mac/Misc/WebDownload.mm:
* Source/WebKitLegacy/mac/Plugins/WebPluginController.mm:
* Source/WebKitLegacy/mac/WebCoreSupport/WebContextMenuClient.mm:
* Source/WebKitLegacy/mac/WebCoreSupport/WebFrameLoaderClient.mm:
* Source/WebKitLegacy/mac/WebView/WebFrame.mm:
* Source/WebKitLegacy/mac/WebView/WebHTMLView.mm:
* Source/WebKitLegacy/mac/WebView/WebImmediateActionController.mm:
* Source/WebKitLegacy/mac/WebView/WebPDFView.mm:
* Source/WebKitLegacy/mac/WebView/WebPreferences.mm:
* Source/WebKitLegacy/mac/WebView/WebPreferencesDefaultValues.mm:
* Source/WebKitLegacy/mac/WebView/WebResource.mm:
* Source/WebKitLegacy/mac/WebView/WebView.mm:
* Source/WebKitLegacy/mac/WebView/WebViewRenderingUpdateScheduler.mm:
* Tools/TestWebKitAPI/Tests/WebCore/cocoa/TestGraphicsContextGLCocoa.mm:
* Tools/TestWebKitAPI/Tests/WebKitCocoa/InAppBrowserPrivacy.mm:
* Tools/TestWebKitAPI/WebCoreTestUtilities.h:
  Fixed imports and namespace qualifiers to reflect new location of RuntimeApplicationChecks.h.

Canonical link: https://commits.webkit.org/284864@main
